vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -133,3 +133,33 @@ enableMarinerKata() {
     # dracut to generate an initrd for the nested VM using binaries from the Mariner host OS.
     systemctlEnableAndStart kata-osbuilder-generate
 }
+
+installFIPS() {
+
+    echo "Installing FIPS..."
+
+    # Install necessary rpm pacakages
+    dnf_install 120 5 25 grubby || exit $ERR_BCC_INSTALL_TIMEOUT
+    dnf_install 120 5 25 dracut-fips || exit $ERR_BCC_INSTALL_TIMEOUT
+    dnf_install 120 5 25 python3-dnf-plugin-versionlock || exit $ERR_BCC_INSTALL_TIMEOUT
```

Comment:
Nit: suggest using ERR_APT_INSTALL_TIMEOUT since this isn't specifically the BCC install timeout. Not that the exit status of installFIPS is checked, though it should be.
```suggestion
    dnf_install 120 5 25 grubby || exit $ERR_APT_INSTALL_TIMEOUT
    dnf_install 120 5 25 dracut-fips || exit $ERR_APT_INSTALL_TIMEOUT
    dnf_install 120 5 25 python3-dnf-plugin-versionlock || exit $ERR_APT_INSTALL_TIMEOUT
```